plone/releaser/manage.py

```diff
@@ -109,10 +113,61 @@ def changelog(**kwargs):
     build_unified_changelog(kwargs["start"], kwargs["end"])
 
 
-def check_checkout(package_name, path):
-    if package_name not in CheckoutsFile(path):
-        msg = "Your package {0} is not on auto-checkout section"
-        raise KeyError(msg.format(package_name))
+def check_checkout(package_name, path=None):
+    """Check if package is in the checkouts.
+
+    If no path is given, we try several paths:
+    both checkouts.cfg and mxdev.ini.
+    """
+    if path:
+        paths = [path]
+    else:
+        paths = glob.glob("mxdev.ini") + glob.glob("checkouts.cfg")
+    for path in paths:
+        if path.endswith(".ini"):
+            checkouts = IniFile(path)
+        else:
+            checkouts = CheckoutsFile(path)
+        if package_name not in checkouts:
+            print(f"No, your package {package_name} is NOT on auto checkout in {path}.")
+        else:
+            print(f"YES, your package {package_name} is on auto checkout in {path}.")
+
+
+def remove_checkout(package_name, path=None):
+    """Remove package from auto checkouts.
+
+    If no path is given, we try several paths:
+    both checkouts.cfg and mxdev.ini.
+    """
+    if path:
+        paths = [path]
+    else:
+        paths = glob.glob("mxdev.ini") + glob.glob("checkouts.cfg")
+    for path in paths:
+        if path.endswith(".ini"):
+            checkouts = IniFile(path)
+        else:
+            checkouts = CheckoutsFile(path)
+        checkouts.remove(package_name)
+
+
+def add_checkout(package_name, path=None):
+    """Add package to auto checkouts.
+
+    If no path is given, we try several paths:
+    both checkouts.cfg and mxdev.ini.
+    """
+    if path:
+        paths = [path]
+    else:
+        paths = glob.glob("mxdev.ini") + glob.glob("checkouts.cfg")
+    for path in paths:
+        if path.endswith(".ini"):
+            checkouts = IniFile(path)
+        else:
+            checkouts = CheckoutsFile(path)
+        checkouts.add(package_name)
```

Comment:
Almost 90% of the `add_checkout`, `remove_checkout` and `check_checkout` code is shared, maybe an auxiliary function to share the code would be good, for future updates? 🍀

Reply:
Done.